pyanaconda/modules/common/structures/subscription.py

```diff
@@ -362,3 +362,107 @@ def server_proxy_password(self) -> SecretData:
     @server_proxy_password.setter
     def server_proxy_password(self, password: SecretData):
         self._server_proxy_password = password
+
+
+class AttachedSubscription(DBusData):
+    """Data for a single attached subscription."""
+
+    def __init__(self):
+        self._name = ""
+        self._service_level = ""
+        self._sku = ""
+        self._contract = ""
+        self._start_date = ""
+        self._end_date = ""
+        self._consumed_entitlement_count = ""
+
+    @property
+    def name(self) -> Str:
+        """Name of the attached subscription.
+
+        :return: subscription name
+        :rtype: str
+        """
+        return self._name
+
+    @name.setter
+    def name(self, name: Str):
+        self._name = name
+
+    @property
+    def service_level(self) -> Str:
+        """Service level of the attached subscription.
```

Comment:
Could you provide an example in the docstring?

Reply:
Added an example.